src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -29,9 +30,14 @@ private ScannedDocumentsHelper() {
 
     @SuppressWarnings("unchecked")
     public static List<Document> getDocuments(CaseDetails caseDetails) {
-        List<Map<String, Object>> data = (List<Map<String, Object>>) caseDetails.getData().get("scannedDocuments");
+        Map<String, Object> caseData = caseDetails.getData();
+        Assert.notNull(caseData, "Case Data must not be null");
 
-        return data.stream()
+        List<Map<String, Object>> scannedDocuments = (List<Map<String, Object>>) caseData.get("scannedDocuments");
+        // ccd returns an empty list when scanned documents are not present.
+        Assert.notNull(scannedDocuments, "Scanned Documents must not be null");
```

Comment:
Should we fail in this case? Or maybe we should set it to null?

Reply:
I'm changing the code to return an empty list